src/main/java/org/gooteam/FortuneTell.java

```diff
@@ -0,0 +1,57 @@
+package org.gooteam;
+
+import net.dv8tion.jda.api.entities.channel.concrete.TextChannel;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import org.gooteam.event.GooMessage;
+import org.gooteam.listener.GooMessageListener;
+import org.jetbrains.annotations.NotNull;
+import java.util.Random;
+
+public class FortuneTell implements GooMessageListener {
+    @Override
+    public void onMessage(@NotNull GooMessage message) {
+        final MessageReceivedEvent event = message.getEvent();
+
+        String FlavorMessage;
+        String ResultMessage;
+
+        String[] FlavorString = {
+                "*Dr Goo farts!*",
+                "*Dr Goo shids!*",
+                "*Dr Goo sharts!*",
+        };
+
+        String[] FortuneString = {
+                "IMMINENT DEMISE",
+                "DISASTER",
+                "BAD LUCK",
+                "NO LUCK",
+                "GOOD LUCK",
+                "GREAT FORTUNE!",
+                "PERFECT ALIGNMENT",
+        };
+
+        String[] FailureString = {
+                "Fortune telling failed! Fuck!",
+                "Epic Fail!",
+                "EPIC FAIL!!!",
+        };
+
+        double FortuneFailChance = Math.random();
```

Comment:
first letter of var names should be lowercase - doesn't matter for this though

Reply:
will keep it in mind